tests/no_version/test_unit_schemas_no_version.py

```diff
@@ -73,14 +73,21 @@ def test_user_create():
     assert u.cache_dir == CACHE_DIR
     # With invalid cache_dir attribute
     with pytest.raises(ValidationError) as e:
-        u = UserCreate(email="a@b.c", password="asd", cache_dir="  ")
+        UserCreate(email="a@b.c", password="asd", cache_dir="  ")
     debug(e.value)
     assert "cannot be empty" in e.value.errors()[0]["msg"]
     # With invalid cache_dir attribute
     with pytest.raises(ValidationError) as e:
-        u = UserCreate(email="a@b.c", password="asd", cache_dir="xxx")
+        UserCreate(email="a@b.c", password="asd", cache_dir="xxx")
     debug(e.value)
     assert "must be an absolute path" in e.value.errors()[0]["msg"]
+    # With invalid cache_dir attribute
+    with pytest.raises(ValidationError) as e:
+        UserCreate(email="a@b.c", password="asd", cache_dir=f"{CACHE_DIR}*;")
+    debug(e.value)
+    assert (
+        "must not contain any of this characters" in e.value.errors()[0]["msg"]
```

Comment:
Let's move to something more readable?
I think we typically either use `"something" in str(e.value)` or `pytest.raises(SomeError, match="something")`

(same comment applies to `tests/v2/04_runner/test_unit_lowlevel_functions.py` below)